as.c

```diff
@@ -270,6 +282,91 @@ void assemble_from_file(vm_env *env, const char *filename)
         assemble_line(env, line);
     }
     free(line);
+}
+
+static inline int mem_not_empty(void *p, int len)
```

Comment:
Can you rewrite this helper function using `strlen`, which is usually optimized for most architectures?